src/query/useGetRpcTxs.ts

```diff
@@ -0,0 +1,50 @@
+import { NetworkHeaders } from "@/types/types";
+import { rpc as StellarRpc } from "@stellar/stellar-sdk";
+
+import { useQuery } from "@tanstack/react-query";
+
+export const useGetRpcTxs = ({
+  rpcUrl,
+  headers,
+  startLedger,
+}: {
+  rpcUrl: string;
+  headers: NetworkHeaders;
+  startLedger: number;
+}) => {
+  const query = useQuery({
+    queryKey: ["getTransactions"],
```

Comment:
We can add `rpcUrl` and `startLedger` for some optimization (caching, for example).

```suggestion
    queryKey: ["useGetRpcTxs", rpcUrl, startLedger],
```